problems/my_merge.rb

```diff
@@ -1,5 +1,5 @@
 class Hash
-  # Write a version of merge. This should NOT modify the original hash and 
+  # Write a `Hash#my_merge(other_hash)` method. This should NOT modify the original hash and 
```

Comment:
Add note similar to above explicitly stating that they are not allowed to use built-in `Hash#merge` method.